### PR TITLE
Comments overhaul

### PR DESCRIPTION
**Pull request by henriquebecker91**
Added comment asked by @blegat. Fixed comment on the negativity/positivity of reduced costs. Added a global const union for a common union. Changed the linebreaks of many comments in an effort to keep them close to 80 characters. Other minor changes.

### CODE REVIEW

File: src/MOI_wrapper.jl [hunk 1]
```diff
@@ -2709,6 +2737,8 @@ function MOI.get(
     model::Optimizer, ::ConstraintConflictStatus,
     index::MOI.ConstraintIndex{
         MOI.ScalarAffineFunction{Float64},
+        # This union does not specify the type parameters of MOI.*
+        # not sure if SUPPORTED_SCALAR_SETS should be used here.
         <:Union{MOI.LessThan, MOI.GreaterThan, MOI.EqualTo}
```
Comment (odow) [inline src/MOI_wrapper.jl:2742]: Fine to use supported sets.

Comment (henriquebecker91) [reply src/MOI_wrapper.jl]: Done,

**Comment by henriquebecker91** [comment]
@odow, I have merged master with this PR. It does not change any behavior, it just improves comments and line breaking. It had conflicts because of changes introduced by my other PR (that solved the buffering bug). Some changes were too close to a comment I changed (or replaced the comment with a more appropriate one), no "real" conflict.

**Comment by odow** [comment]
Screenshot of tests passing?

**Comment by henriquebecker91** [comment]
[](https://postimg.cc/gXGs2qZ8)

**Comment by odow** [comment]
Thanks

**Pull request merged**